upgrade_tests/test_existance_relations/test_filter.py

```diff
@@ -40,7 +41,7 @@ def test_positive_filters_by_resource_type(pre, post):
     :expectedresults: All filters of all roles should be retained post upgrade
         by resource types
     """
-    assert assert_varients(component, pre, post)
+    assert post == property_name_on_post_version(component, pre)
```

Comment:
I chose this approach to been able to know what os going on only by looking the test. In your approach I had to really look into the function and I took some time to understand it, once the lookup on the matrix was done by numeric indexes.